### DISCUSSION TITLE
Dev5.1.04081

### DISCUSSION THREAD

**Discussion by Happy2018new** [rnhws-Team/PhoenixBuilder-Achieved#107] [General]
<h1>注意: Dev预览版仅支持Linux,Android,Windows平台</h1>

## What's Changed

* https://github.com/rnhws-Team/PhoenixBuilder/pull/106 by @Happy2018new in https://github.com/LNSSPsd/PhoenixBuilder/pull/273

**Full Changelog**: https://github.com/rnhws-Team/PhoenixBuilder/compare/Dev5.1.04080...Dev5.1.04081

And a special thanks to @CMA2401PT @LNSSPsd @Torrekie

<hr /><em>This discussion was created from the release <a href='https://github.com/rnhws-Team/PhoenixBuilder/releases/tag/Dev5.1.04081'>Dev5.1.04081</a>.</em>

**Comment by Torrekie** [comment]
Please stop @'ing developers, we continuously receive those emails which really confusing. You can change this special thanks to a reference to upstream repo

**Reply by Happy2018new** [reply]
> Please stop @'ing developers, we continuously receive those emails which really confusing. You can change this special thanks to a reference to upstream repo

Oh, I'm sorry about this. We will take other measures to solve the problem soon. Sorry!

**Comment by Mono2023-PRC** [comment]
#106 
**并不能下载对应的文件**
![image](https://user-images.githubusercontent.com/109840665/232177878-e49368cb-e8d2-4207-b1af-b8a4d7a9eef1.png)
![image](https://user-images.githubusercontent.com/109840665/232177887-d30f6d7f-4077-462a-adac-074a3529b44f.png)
![image](https://user-images.githubusercontent.com/109840665/232178029-206f24bc-c2c7-4bde-a992-ad9d4aecd790.png)

**Reply by Happy2018new** [reply]
恐怕我们得自己搭建一个同样的站点来传输这些文件才行，但看起来有些麻烦，不如让使用者自己从存储库取吧。
***
<img width="960" alt="image" src="https://user-images.githubusercontent.com/109064184/232199238-302fd7b7-6382-4111-8b5e-81cab1b8b49a.png">
